### PR TITLE
wgsl: allow unsigned workgroup_size components

### PR DESCRIPTION
**Pull request by dneto0**
Fixes: #1857

### CODE REVIEW

File: wgsl/index.bs [hunk 1]
```diff
@@ -5426,7 +5426,7 @@ ISSUE: Can we query upper bounds on workgroup size dimensions?  Is it independen
 
 <div class='example wgsl global-scope' heading='workgroup_size Attribute'>
   <xmp>
-    [[ stage(compute), workgroup_size(8,1,1) ]]
+    [[ stage(compute), workgroup_size(8,4u,1) ]]
```
Comment (alan-baker) [inline wgsl/index.bs:5429]: Not all components are the same type in this example

**Review by alan-baker — CHANGES_REQUESTED** [review]

File: wgsl/index.bs (hunk 1 above)
Comment (dneto0) [reply wgsl/index.bs]: Fixed.  Thanks.

File: wgsl/index.bs [hunk 2]
```diff
@@ -5426,16 +5426,25 @@ ISSUE: Can we query upper bounds on workgroup size dimensions?  Is it independen
 
 <div class='example wgsl global-scope' heading='workgroup_size Attribute'>
   <xmp>
-    [[ stage(compute), workgroup_size(8,1,1) ]]
+    [[ stage(compute), workgroup_size(8,4,1) ]]
     fn sorter() { }
        // OpEntryPoint GLCompute %sorter "sorter"
        // OpExecutionMode %sorter LocalSize 8 1 1
```
Comment (alan-baker) [inline wgsl/index.bs:5432]: This dimension no longer matches.

**Review by alan-baker — CHANGES_REQUESTED** [review]

**Review by kvark — APPROVED** [review]

**Review by alan-baker — APPROVED** [review]

**Comment by jdashg** [comment]
<details><summary><a href="https://github.com/gpuweb/gpuweb/wiki/WGSL-2021-06-22-Minutes#wgsl-allow-unsigned-workgroup_size-components-1858">WGSL meeting minutes 2021-06-22</a></summary>



*   (but require them to be all the same type)
*   (consensus: merge)


</details>

**Pull request merged**